src/ast.cpp

```diff
@@ -72,6 +72,13 @@ void IfStmtNode::Accept(ModifyingVisitor& v) {
   v.Visit(*this);
 }
 
+void WhileStmtNode::Accept(NonModifyingVisitor& v) const {
+  v.Visit(*this);
+}
+
+void WhileStmtNode::Accept(ModifyingVisitor& v) {
+  v.Visit(*this);
+}
```

Comment:
```suggestion
}

```
Insert a blank line between functions. Unfortunately, the formatter doesn't handle this for us. :disappointed: